Reuse Gcode M123 in Prusa "FAN" command

gudnimg opened the pull request:
Changes save 90 bytes of flash

The `FAN` prusa command now not only shows RPM for both fans but also the fan speed (same as Gcode M123). I'm not sure if the format of the message breaks anything internally at Prusa. Please let me know. I mainly want to reduce the occurrences of `fan_speed[]` in the code.

gudnimg committed:
Reuse Gcode M123 in Prusa command 'FAN'

Saves 90 bytes of flash memory

leptun commented:
@TojikCZ can answer your question about compatibility

leptun commented:
As for the PSTR change, I already fixed it in 3.10.2 or in 3.11 (don't remember). Since that change already is there, this will interfere when we merge those branches to MK3 (soon™)

gudnimg commented:
> As for the PSTR change, I already fixed it in 3.10.2 or in 3.11 (don't remember). Since that change already is there, this will interfere when we merge those branches to MK3 (soon™)

Nice, I've removed the PSTR commit.

gudnimg commented:
Added another commit where we can reuse M123.

<img width="297" alt="image" src="https://user-images.githubusercontent.com/8218499/156630815-b4fe7047-ff03-4b53-8aa1-a487ba76fd1e.png">

wavexx commented:
Huh, just touched that part in #3461. We can definitely replace the code in selftest (I will probably do the same already, since it would conflict anyway).

No idea about the "FAN" command though.. something else might rely on this

gudnimg commented:
I've updated the PR by removing the 2nd commit for fan selftest. Changes now only affect Prusa "FAN" command.
